wiki/Tournaments/OZT/2018/en.md

```diff
@@ -0,0 +1,114 @@
+# osu! Z-Tournament 2018
+![osu!Z-Tournament 2018](logo.png)
+
+- The **osu! Z-Tournament 2018**, is a 4v4 international osu!standard tournament.
+- There are no rank limits, therefore allowing any player to join.
+- Score system: **Scorev2**
+- This tournament will hold up to **64 teams, with 6 players at minimum and 8 players at maximum in each team.**
+- **Team admission will be decided through their PP average**, which means that the teams with the highest PP average will take priority.
```

Comment:
This looks like it should be in a "tournament registration" or "tournament rules" section.